doc/index.rst

```diff
@@ -1,8 +1,45 @@
-Plutus
-======
+Plutus Programming and Reference Guide
+=============================================
+
+About Plutus
+--------------------
+
+Plutus consists of Plutus Core, the programming, scripting and assembly language; 
+and Plutus Tx, the compiler that compiles the Haskell source code into Plutus Core 
+to form the on-chain part of a contract application. All of this is used in combination 
+to write Plutus scripts that run on the Cardano blockchain. Compiled Plutus Core 
+scripts or programs are able to interact with the Cardano ledger through the 
+ledger interface. 
```

Comment:
When we talk about the "ledger interface" we usually mean the interface that _we_ provide to the ledger, not the interface that the ledger provides to scripts.

Reply:
That's helpful to know. I've updated this statement to look like this: 

"The Cardano ledger is able to interact with compiled Plutus Core scripts or programs through the ledger interface."

Reply:
Also I'm not sure whether the target audience actually needs to know about that.